docassemble/ALAutomatedTestingTests/data/questions/al_set_up_testing.yml

```
@@ -511,10 +524,22 @@ subquestion: |
   
   If you want to make one later instead, save this link to [our documentation](https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/automated_integrated_testing).
 fields:
-  - Which interview files do you want the test to use?: installer.test_files_wanted
+  - Do you have more than one YAML file that starts an interview?: more_than_one_file
+    datatype: yesnoradio
+  - Which of those interview files do you want to run in these tests?: multiple_files
     datatype: checkboxes
     code: |
       installer.get_question_file_names()
+    show if: more_than_one_file
+  - Which interview file do you want to run in this test?: single_file
+    input type: radio
+    # A field can only have one type, so the answers all have to be strings
+    # Hense 'None' as a value instead of `None`
+    code: |
+      installer.get_question_file_names() + [[ 'None', 'None' ]]
```

Is might get confusing fast; how would you feel about `"None of the Above"`, or `"None of these"` as the string to check instead?

---

```suggestion
      installer.get_question_file_names() + [[ 'None', "Don't make any new file" ]]
```

```suggestion
      installer.get_question_file_names() + [[ 'None', "Skip this" ]]
```

```suggestion
      installer.get_question_file_names() + [[ 'None', "Don't create any tests" ]]
```

Apparent choice paralysis. Do any of these spark your fancy?

---

Skip this is fine